Algorithm.Python/CustomDataBitcoinAlgorithm.py

```diff
@@ -86,7 +86,7 @@ def Reader(self, config, line, date, isLiveMode):
                 value = liveBTC["last"]
                 if value == 0: return None
 
-                coin.Time = datetime.now()
+                coin.Time = datetime.now() - timedelta(hours=4)
```

Comment:
Lean expects all data to be in exchange time zone, believe the C# original implementation is correct?
`datetime.now()` returns local time zone, same for `DateTime.Now`, should probably be using `datetime.utcnow()` and converting into the exchange timezone as expected by Lean, else the result will depend on the local machine timezone